pwa/src/components/entities/entitiesTable.tsx

```diff
@@ -96,8 +96,8 @@ export default function EntitiesTable() {
                         field: "endpoint",
                       },
                       {
-                        headerName: "Route",
-                        field: "route",
+                        headerName: "Path",
+                        field: "path",
```

Comment:
Je kan `field` niet zomaar aanpassen van `route` naar `path` omdat dit inhaakt op een object die `route` verwacht (dit krijgen we namelijk als response vanuit de gateway). 

Dit betekent dat deze tabel nu nooit meer een `route` kan tonen. De oplossing hiervoor is dat je de `headerName` naar _Path_ hernoemd, terwijl je de `field` onaangepast laat.

Zie hieronder het object wat we terug krijgen:

```
{
...
  dateCreated: null
  dateModified: null
  description: "test"
  endpoint: "test"
  extend: false
  function: "organization"
  gateway: null
  id: "54a77618-fc9b-43bb-8603-f3ee0e48a696"
  inherited: false
  name: "test"
  route: "test"
...
}
```

Reply:
Zie in bovenstaand object dat er een key `route` in staat die we dus moeten aanspreken in de code.